Android packages build error

```
 [1486/1667] Linking target test/unit/auxiliary/subprocess-helloworld
clang: warning: argument unused during compilation: '-pie' [-Wunused-command-line-argument]
[1487/1667] Compiling C object binrz/rz-test/rz-test.p/rz-test.c.o
../binrz/rz-test/rz-test.c:312:24: warning: suggest braces around initialization of subobject [-Wmissing-braces]
        RzTestState state = { 0 };
                              ^
                              {}
1 warning generated.
 ../test/unit/test_analysis_block.c:60:28: warning: suggest braces around initialization of subobject [-Wmissing-braces]
        RzAnalysisBlock dummy = { 0 };
                                  ^
                                  {}
../test/unit/test_analysis_block.c:60:28: warning: suggest braces around initialization of subobject [-Wmissing-braces]
        RzAnalysisBlock dummy = { 0 };
                                  ^
                                  {}
2 warnings generated.
 [1508/1667] Compiling C object librz/core/librz_core.a.p/cmd.c.o
../librz/core/cmd.c:1730:20: warning: function 'print_dict' is not needed and will not be emitted [-Wunneeded-internal-declaration]
static inline void print_dict(RzCoreAutocomplete *a, int sub) {
                   ^
1 warning generated.
../binrz/blob/main.c:16:2: warning: implicit declaration of function 'UTF8_ARGV_FREE' is invalid in C99 [-Wimplicit-function-declaration]
        UTF8_ARGV_FREE(argc, utf8_argv);
        ^
1 warning generated.
[1481/1667] Compiling C object binrz/rz-test/rz-test.p/load.c.o
[1482/1667] Compiling C object binrz/rz-test/rz-test.p/rz-test.c.o
../binrz/rz-test/rz-test.c:312:24: warning: suggest braces around initialization of subobject [-Wmissing-braces]
        RzTestState state = { 0 };
                              ^
                              {}
1 warning generated.
usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/armv7a-linux-androideabi28-clang  -o binrz/blob/rizin binrz/blob/rizin.p/main.c.o -Wl,--as-needed -Wl,--no-undefined -Wl,-O1 -pie -static -Wl,--start-group librz/main/librz_main.a librz/util/librz_util.a subprojects/sdb/src/libsdb.a subprojects/zlib-1.2.11/libz.a librz/magic/librz_magic.a librz/socket/librz_socket.a librz/flag/librz_flag.a librz/cons/librz_cons.a librz/hash/librz_hash.a subprojects/xxhash/libxxhash.a librz/crypto/librz_crypto.a librz/io/librz_io.a shlr/librzbochs.a shlr/gdb/libr2gdb.a shlr/librzwinkd.a shlr/librzqnx.a subprojects/libzip-1.7.3/liblibzip.a shlr/librzar.a librz/reg/librz_reg.a librz/bp/librz_bp.a librz/syscall/librz_syscall.a librz/parse/librz_parse.a librz/asm/librz_asm.a librz/bin/librz_bin.a subprojects/lz4-1.9.3/liblz4.a subprojects/yxml/libyxml.a subprojects/capstone-bundled/libcapstone.a librzspp.a librz/egg/librz_egg.a subprojects/mpc/libmpc.a librz/search/librz_search.a librz/analysis/librz_analysis.a librz/diff/librz_diff.a librz/type/librz_type.a subprojects/tree-sitter-0.19.4/libtree_sitter.a subprojects/tree-sitter-c/libtree-sitter-c.a librz/debug/librz_debug.a librz/config/librz_config.a librz/core/librz_core.a librz/lang/librz_lang.a shlr/rizin-shell-parser/libshell_parser.a subprojects/libuv-v1.40.0/libuv.a -ldl -lm -Wl,--end-group -pthread
clang: warning: argument unused during compilation: '-pie' [-Wunused-command-line-argument]
binrz/blob/rizin.p/main.c.o:main.c:function main: error: undefined reference to 'UTF8_ARGV_FREE'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
[1519/1667] Linking target test/unit/test_analysis_hints
clang: warning: argument unused during compilation: '-pie' [-Wunused-command-line-argument]
[1520/1667] Linking target test/unit/test_agraph
clang: warning: argument unused during compilation: '-pie' [-Wunused-command-line-argument]
ninja: build stopped: subcommand failed.
Error: Process completed with exit code 1.
```

--- a/binrz/blob/main.c
+++ b/binrz/blob/main.c
@@ -13,6 +13,6 @@ int MAIN_NAME(int argc, const ARGV_TYPE **argv) {
 		rc = rz_main_run(m, argc, (const char **)utf8_argv);
 		rz_main_free(m);
 	}
-	UTF8_ARGV_FREE(argc, utf8_argv);
+	FREE_UTF8_ARGV(argc, utf8_argv);
 	return rc;
 }

